app/views/candidate_interface/satisfaction_survey/adaptability.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :title, title_with_error_prefix(t('page_titles.adaptability'), @survey.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_satisfaction_survey_consistency_path) %>
+
+<%= form_with model: @survey, url: candidate_interface_satisfaction_survey_submit_adaptability_path, method: :post do |f| %>
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h2 class="govuk-heading-xl">
+        <%= t('page_titles.adaptability') %>
+      </h2>
+
+      <%= render 'form', f: f %>
+    </div>
+  </div>
+<% end %>
```

Comment:
Shall we turn this whole thing into a partial?

Thinking this could work:

```ruby
<%= render 'survey_page',
  title: t('page_titles.adaptability'),
  url: candidate_interface_satisfaction_survey_submit_adaptability_path
%>
```